tests/Unit/Evolution/Initialization/Test_ConservativeSystem.cpp

```diff
@@ -89,9 +91,10 @@ void check_primitives(std::true_type /*has_prims*/, const Runner& runner,
   using prim_vars_tag = Tags::Variables<tmpl::list<PrimVar>>;
   CHECK(ActionTesting::get_databox_tag<comp, prim_vars_tag>(runner, 0)
             .number_of_grid_points() == number_of_grid_points);
-  CHECK(ActionTesting::get_databox_tag<
-            comp, typename Metavariables<Dim, true>::equation_of_state_tag>(
-            runner, 0) == 5);
+  // TODO :  no generic way to compare EoS
+  // CHECK(ActionTesting::get_databox_tag<
```

Comment:
If you know the EOS you can do something like `dynamic_cast<const EquationOfState::PolytropicFluid&>(get_databox_tag...)`